docs/admin-guide/loading-case.md

```diff
@@ -75,6 +75,8 @@ Available types or case reports:
 - **delivery**: Delivery Report (HTML)
 - **cnv**: Copy Number Variants Report (PDF), available only for cancer cases
 - **cov_qc**: Coverage QC Report (HTML), available only for cancer cases
+- **exe_ver**: Pipeline detailed software versions (YAML)
+- **reference_info**: Pipeline detailed reference file versions (YAML)
```

Comment:
This key could also be abbreviated to `ref_info`, but it's fine also like this

Reply:
It just feels easiest to keep it consistent all the way through MIP-deliverables/Hermes/Housekeeper/CG/Scout, though CG/Hermes could certainly map it for us.

Reply:
No, but then it's OK